community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelStatement.java

```diff
@@ -187,14 +207,35 @@ final void forceClose()
         }
     }
 
-    private void cleanupResources()
+    final String authSubjectName()
     {
-        storeStatement.release();
-        // closing is done by KTI
+        return transaction.mode().name();
```

Comment:
find it a bit strange how mode().name and authSubject relates to each other and how we use String to represent it for some reason